### PR TITLE
feature/about us page

### PR DESCRIPTION
**Pull request by Pakholchuk-Oleh**
https://trello.com/c/SuyztAYY/34-create-page-about-us
- [x] add style
- [x] google maps
- [x] view
- [ ] icons
- [x] slider
- [ ] mobile version

### CODE REVIEW

File: app/assets/stylesheets/about.scss [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        width: 290px;
+        height: 60px;
+        top: 385px;
+        background-color: #000000;
+        border-radius: 20px 20px 0px 0px;
+      }
+    }
+
+  }
+}
```
Comment (viktoriia-horobets) [inline app/assets/stylesheets/about.scss:94]: add line

File: app/views/home/about.html.erb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  <div>
+    <h5>Ми працюємо тут</h5>
+    <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2643.8578527847394!2d35.07007641566177!3d48.49761767925375!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x40d95879f91baeff%3A0x7a056632b3e46b5e!2zU3RvbGpldG92YSBTdCwgMjEsINCU0L3RltC_0YDQvsyBLCBEbmlwcm9wZXRyb3ZzJ2thIG9ibGFzdCwgNDkwMDA!5e0!3m2!1sen!2sua!4v1662767399591!5m2!1sen!2sua" class="google-map" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
+  </div>
+  
+  <div class="collage">
+    <h6>Дніпровська міська рада</h6>
+
+  </div>
+</div>
```
Comment (viktoriia-horobets) [inline app/views/home/about.html.erb:58]: add line

File: app/views/home/about.html.erb [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    </div>
+  </div>
+  
+  <div>
+    <h5>Ми працюємо тут</h5>
+    <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2643.8578527847394!2d35.07007641566177!3d48.49761767925375!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x40d95879f91baeff%3A0x7a056632b3e46b5e!2zU3RvbGpldG92YSBTdCwgMjEsINCU0L3RltC_0YDQvsyBLCBEbmlwcm9wZXRyb3ZzJ2thIG9ibGFzdCwgNDkwMDA!5e0!3m2!1sen!2sua!4v1662767399591!5m2!1sen!2sua" class="google-map" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
+  </div>
+  
+  <div class="collage">
+    <h6>Дніпровська міська рада</h6>
```
Comment (viktoriia-horobets) [inline app/views/home/about.html.erb:55]: please, use i18n

File: app/views/home/about.html.erb [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+          <p class="text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloremque necessitatibus nisi iste neque aspernatur officiis consequuntur.</p>
+          <p>test@gmail.com</p>
+          <p>+380778772409</p>
+        </div>
+      </div>
+    </div>
+  </div>
+  
+  <div>
+    <h5>Ми працюємо тут</h5>
```
Comment (viktoriia-horobets) [inline app/views/home/about.html.erb:50]: please, use i18n

File: app/assets/stylesheets/about.scss (hunk 1 above)
Comment (Pakholchuk-Oleh) [reply app/assets/stylesheets/about.scss]: fixed

File: app/views/home/about.html.erb (hunk 2 above)
Comment (Pakholchuk-Oleh) [reply app/views/home/about.html.erb]: fixed

File: app/views/home/about.html.erb (hunk 3 above)
Comment (Pakholchuk-Oleh) [reply app/views/home/about.html.erb]: fixed

File: app/views/home/about.html.erb (hunk 4 above)
Comment (Pakholchuk-Oleh) [reply app/views/home/about.html.erb]: fixed

File: app/javascript/packs/scripts/team-slider.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
+const PREV  = document.querySelector('.prev'),
```
Comment (mykhailokonst) [inline app/javascript/packs/scripts/team-slider.js:1]: const previousValue  = document.querySelector('.prev');

File: app/javascript/packs/scripts/team-slider.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+let sliderWidth = document.querySelector('.slider-container').offsetWidth;
+
+window.addEventListener('resize', () => {
+  sliderWidth = document.querySelector('.slider-container').offsetWidth;
+})
+
+let index = 0;
+
+NEXT.addEventListener('click', () => {
```
Comment (mykhailokonst) [inline app/javascript/packs/scripts/team-slider.js:13]: nextValue

File: app/javascript/packs/scripts/team-slider.js [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+  index++;
+  PREV.classList.add('show');
```
Comment (mykhailokonst) [inline app/javascript/packs/scripts/team-slider.js:15]: previousValue

File: app/javascript/packs/scripts/team-slider.js [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+  TRACK.style.transform = `translateX(-${index * sliderWidth}px)`;
```
Comment (mykhailokonst) [inline app/javascript/packs/scripts/team-slider.js:16]: track

**Review by mykhailokonst — CHANGES_REQUESTED** [review]

**Pull request merged**